### PR TITLE
Fix tutorial 11

### PR DESCRIPTION
**Pull request by mitschabaude**
some changes to https://github.com/o1-labs/docs2/pull/348 to make it work with the current version of snarkyjs

### CODE REVIEW

File: docs/zkapps/tutorials/11-advanced-account-updates.mdx [hunk 1]
```diff
@@ -23,7 +23,6 @@ This tutorial was last tested with [SnarkyJS](https://www.npmjs.com/package/snar
 
 ## Overview:
```
Comment (barriebyron) [inline docs/zkapps/tutorials/11-advanced-account-updates.mdx:24]:
```suggestion
## Overview
```

**Review by barriebyron — APPROVED** [review]
a teensy suggestion, thanks for this improvement

File: docs/zkapps/tutorials/11-advanced-account-updates.mdx [hunk 2]
```diff
@@ -82,10 +84,9 @@ The `myTokenInstance.approveDeploy(...)` code is as follows:
 
 ```typescript
   @method approveDeploy(deployUpdate: AccountUpdate) {
-    // TODO should anything else be asserted here for a "safe" deployment?
```
[the remark this answers is not in the document]
Comment (mitschabaude) [reply docs/zkapps/tutorials/11-advanced-account-updates.mdx:85]: NoChildren ensures that the account update we're approving cannot have children of its own. Which means we only have to check this one account update.

The possible side effects still include everything that this one account update can do to its account. It's application-specific what part of token accounts the token manager wants to control. In this case, we only want to control the balance, so the balance check below is sufficient.

File: docs/zkapps/tutorials/11-advanced-account-updates.mdx [hunk 3]
```diff
@@ -96,66 +97,62 @@ This checks that the deployUpdate is a single account update, with no children -
...
     this.self.body.mayUseToken = AccountUpdate.MayUseToken.ParentsOwnToken;
   }
 ```
 
 And is approved by the token contract's `approveTransfer`:
 
 ```typescript
-  @method approveTransfer(
-    transferUpdate: AccountUpdate
-  ) {
```
[the remark this answers is not in the document]
Comment (mitschabaude) [reply docs/zkapps/tutorials/11-advanced-account-updates.mdx:141]: Same answer as above https://github.com/o1-labs/docs2/pull/409#discussion_r1236426784

**Pull request merged**